src/core/agent/services/connectionService.ts

```diff
@@ -397,6 +397,10 @@ class ConnectionService extends AgentService {
     });
   }
 
+  async getKeriContacts(): Promise<KeriContact[]> {
```

Comment:
Where is this used?

Reply:
I am planning to use this in the screen where the creator chooses other accounts before creating the multisig.

Reply:
Would it be better if we just fetched the metadata records, so it matches the list in the UI?

Reply:
Yeah, that would be better. But currently the function`getAllConnectionKeriMetadata` is still private. I will remove `getKeriContacts` and make the `getAllConnectionKeriMetadata` public. What do you think?

Reply:
Works for me yes :)

Reply:
`getKeriContacts` is now unused and can be deleted